swagger.yaml

```diff
@@ -863,6 +863,285 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+    
+  /reservation/:
+    post:
+      summary: 予約申し込み
+      tags:
+      - reservation
+      requestBody:
+        description: 予約するための個人情報
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/reservationInfo"
+      responses:
+        200:
+          description: |
+            予約ID  
+            またメールにメール認証用のコードを送付
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  reservation_id:
+                    type: string
+  /reservation/auth_email/:
+    post:
+      summary: メール認証
+      tags:
+      - reservation
+      requestBody:
+        description: メール認証用の情報
+        content:
+          application/json:
+            schema:
+              allOf:
+                - $ref: "#/components/schemas/reservationInfo"
+                - properties:
+                    email:
+                      type: string
+                      writeOnly: false
```

Comment:
```suggestion
              type: object
              properties:
                auth_id:
                  type: string
                  writeOnly: false
```

Reply:
fix: now request the only auth_id in reservation_auth